types/tests/index.test-d.ts

```diff
@@ -686,14 +703,18 @@ function text(this: NightwatchAssertion<string>, selector: string, expectedText:
   this.value = (result) => result.value!;
 
   // TODO: fix callback types
```

Comment:
we can remove TODO comment, if the issue is resolved

Reply:
I have a few changes in that file. Will make this change in the next PR.